core/src/main/java/com/novoda/noplayer/internal/exoplayer/forwarder/VideoRendererInfoForwarder.java

```diff
@@ -1,91 +0,0 @@
-package com.novoda.noplayer.internal.exoplayer.forwarder;
-
-import android.view.Surface;
-
-import com.google.android.exoplayer2.Format;
-import com.google.android.exoplayer2.decoder.DecoderCounters;
-import com.google.android.exoplayer2.video.VideoRendererEventListener;
-import com.novoda.noplayer.NoPlayer;
-
-import java.util.HashMap;
-
-import static com.novoda.noplayer.internal.exoplayer.forwarder.ForwarderInformation.Methods;
-import static com.novoda.noplayer.internal.exoplayer.forwarder.ForwarderInformation.Parameters;
-
-class VideoRendererInfoForwarder implements VideoRendererEventListener {
-
-    private final NoPlayer.InfoListener infoListener;
-
-    VideoRendererInfoForwarder(NoPlayer.InfoListener infoListener) {
-        this.infoListener = infoListener;
-    }
-
-    @Override
-    public void onDroppedFrames(int count, long elapsedMs) {
-        HashMap<String, String> callingMethodParameters = new HashMap<>();
-
-        callingMethodParameters.put(Parameters.COUNT, String.valueOf(count));
-        callingMethodParameters.put(Parameters.ELAPSED_MS, String.valueOf(elapsedMs));
-
-        infoListener.onNewInfo(Methods.ON_DROPPED_VIDEO_FRAMES, callingMethodParameters);
-    }
-
-    @Override
-    public void onVideoEnabled(DecoderCounters counters) {
-        HashMap<String, String> callingMethodParameters = new HashMap<>();
-
-        callingMethodParameters.put(Parameters.DECODER_COUNTERS, String.valueOf(counters));
-
-        infoListener.onNewInfo(Methods.ON_VIDEO_ENABLED, callingMethodParameters);
-    }
-
-    @Override
-    public void onVideoDecoderInitialized(String decoderName, long initializedTimestampMs, long initializationDurationMs) {
```

Comment:
we do not have this callback anymore? and others...

Reply:
These are exposed through the `analytics listener` now.